modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -52,133 +86,174 @@ handled by the codemod are marked with 🤖 in the Upgrade Guide.**
 the changes from the codemod as a single isolated commit (separate from other changes) so you can
 roll back more easily if necessary.
 
-## Component Deprecations
+## Token Updates
+
+### Depth
 
-### Deprecation Types
+**PR:** [#2091](https://github.com/Workday/canvas-kit/pull/2091)
+
+In v7, we released an update to our depth tokens that was too bold and harsh for web applications.
+We've modified the depth token styles to be more subtle and improve visual design. This change
+affects all components which use depth tokens including `Card`, `Toast`, `Dialog`, `Popup`, `Modal`,
+and `Menu`. We have not changed which depth values each component references (i.e., `Card` remains
+at `depth[1]`).
 
-#### Soft Deprecation
+## Component Removals
```

Comment:
I've removed the `Deprecation Types`, `Soft Deprecation`, and `Hard Deprecation` sections of the upgrade guide. We'll use `Component Removals` and `Component Deprecations` going forward as per #2042 (though I'm now wondering if `Removed Components` and `Deprecated Components` would serve as better headings 🤔).

Note that no components were deprecated in v9 so there is no `Component Deprecations` section here.

Reply:
Good call on the potential for utility deprecations (there's even one already in the Upgrade Guide with `memoizedFocusRing`). Thinking about going with a `Deprecations and Removals` section.